client/src/main/java/org/evosuite/junit/JUnit5TestAdapter.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (C) 2010-2018 Gordon Fraser, Andrea Arcuri and EvoSuite
+ * contributors
+ *
+ * This file is part of EvoSuite.
+ *
+ * EvoSuite is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3.0 of the License, or
+ * (at your option) any later version.
+ *
+ * EvoSuite is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with EvoSuite. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.evosuite.junit;
+
+import org.evosuite.Properties;
+import org.evosuite.junit.writer.TestSuiteWriterUtils;
+import org.evosuite.runtime.vnet.NonFunctionalRequirementExtension;
+import org.evosuite.testcase.TestCase;
+import org.evosuite.testcase.TestCodeVisitor;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * <p>
+ * JUnit4TestAdapter class.
```

Comment:
Just a minor detail, but it seems it should read "JUnit5TestAdapter" instead of "JUnit4TestAdapter"?